packages/appium/lib/main.js

```diff
@@ -12,13 +13,12 @@ import { AppiumDriver } from './appium';
 import { driverConfig, pluginConfig, USE_ALL_PLUGINS } from './cli/args';
 import { runExtensionCommand } from './cli/extension';
 import { default as getParser, SERVER_SUBCOMMAND } from './cli/parser';
-import { APPIUM_VER, checkNodeOk, getGitRev, getNonDefaultServerArgs, showConfig, validateTmpDir, warnNodeDeprecations } from './config';
+import { APPIUM_VER, checkNodeOk, getGitRev, getNonDefaultServerArgs, showBuildInfo, validateTmpDir, warnNodeDeprecations, showConfig } from './config';
```

Comment:
this line seems too long to me

Reply:
imo if it didn't fail eslint it's not worth fiddling with.  if we need formatting, `prettier` does a good job

Reply:
It is hard to read such code. We write code for people, not for linters